File: 1893 check-if-all-the-integers-in-a-range-are-covered/solution.java
class Solution {
    public boolean isCovered(int[][] ranges, int left, int right) {
        for(int s=left;s<=right;s++) {
            boolean cov=false;
            for(int[] r:ranges) {
                if(r[0]<=s && s<=r[1]) cov=true;
            }
            if(!cov) return false;
        }
        return true;
    }
}

File: 1893 check-if-all-the-integers-in-a-range-are-covered/solution.go
func isCovered(ranges [][]int, left int, right int) bool {
    for i:=left;i<=right;i++ {
        cov:=false
        for _,v:= range ranges {
            if v[0]<=i && i<=v[1] {
                cov=true
            }
        }
        if !cov {
            return false
        }
    }
    return true
}
